src/Compilers/CSharp/Portable/Syntax/SyntaxEquivalence.cs

```diff
@@ -82,6 +82,8 @@ private static bool AreTokensEquivalent(GreenNode? before, GreenNode? after, Fun
                 case SyntaxKind.NumericLiteralToken:
                 case SyntaxKind.CharacterLiteralToken:
                 case SyntaxKind.StringLiteralToken:
+                case SyntaxKind.SingleLineRawStringLiteralToken:
```

Comment:
Do we have tests affected by this change?

Reply:
i'll check.

Reply:
Gentle ping

Reply:
Did you find out if tests are affected by this change?